src/util/triplea/MapXMLCreator/ImageScrollPanePanel.java

```diff
@@ -0,0 +1,309 @@
+package util.triplea.MapXMLCreator;
+
+import games.strategy.util.PointFileReaderWriter;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Cursor;
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.HeadlessException;
+import java.awt.Image;
+import java.awt.Point;
+import java.awt.Polygon;
+import java.awt.Rectangle;
+import java.awt.Toolkit;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.geom.Rectangle2D;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JViewport;
+import javax.swing.SwingUtilities;
+
+import util.image.FileOpen;
+
+/**
+ * Base class for image *Panel classes that show a map for defining certain map XML properties.
+ * It contains the polygons and center points of the territories.
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+public abstract class ImageScrollPanePanel
+{
+	private static Font s_font = null;
+	
+	protected static MapXMLCreator s_mapXMLCreator;
+	protected static Map<String, List<Polygon>> s_polygons = new HashMap<String, List<Polygon>>(); // hash map for polygon points
+	public static boolean s_polygonsInvalid = true;
+	
+	private JPanel m_imagePanel;
+	
+	protected void layout(final JPanel stepActionPanel)
+	{
+		final Dimension size = stepActionPanel.getSize();
+		JScrollPane js = new JScrollPane(createImagePanel());
+		js.setBorder(null);
+		stepActionPanel.setLayout(new BorderLayout());
+		stepActionPanel.add(js, BorderLayout.CENTER);
+		stepActionPanel.setPreferredSize(size);
+	}
+	
+	protected static ImageScrollPanePanel s_instance;

```

Comment:
will remove the variable s_instance all together. I think I introduced it to save rebuilding panels, but changed the logic again as some cross-panel-dependencies and the possibility to jump between the steps the panels belong to.